### ISSUE TITLE
so when are we building this treefort

### ISSUE THREAD

**Issue by junosuarez** [treefortPDX/treefortpdx.github.com#2]
🌲🌲🌲🌲🌲🌲🌲🌲🌲🌲🌲🌲🌲🌲🌲🌲🌲🌲

**Comment by terinjokes** [comment]
I'll be in Portland starting on December 7th to December 13th (for Node Interactive). :smile:

**Comment by junosuarez** [comment]
I, too, will be at Node Interactive

**Comment by aulvi** [comment]
I'm not scheduled to attend Node Interactive but should be around for drinking and other merry-making!

**Comment by aulvi** [comment]
(ps, I have a couple of small chainsaws)

**Comment by junosuarez** [comment]
> chainsaws

:+1:

**Comment by junosuarez** [comment]
:evergreen_tree:

**Comment by terinjokes** [comment]
I will no longer be in town for Node Interactive.

**Comment by aulvi** [comment]
Bummer, sorry you can't make it.

In other news, I have an extra saw available!

**Comment by thoward** [comment]
Well, we'll need a new site for the treefort, right? Since @nvcexploder is off in CA now... I don't have a suitable tree in my yard. Maybe @mattly's place?

**Comment by nvcexploder** [comment]
We still have the same house! We'll be back there in May ... next summer
may be a good time to do this :D

**Comment by aulvi** [comment]
@nvcexploder Right on!

(ps update your github profile)

**Comment by aulvi** [comment]
@nvcexploder Any plans for treefort this coming May?

**Comment by nvcexploder** [comment]
No, we're selling the house this summer. :(

**Comment by aulvi** [comment]
@nvcexploder Please let me know when it's listed, we are looking for a larger place and some land but are unsure where.

We should totally drop a tree and split it into boards before you sell the place though. Could be a while until you have a personal forest again.

**Comment by nvcexploder** [comment]
Definitely!!! We'll be up there in May. I can always call you before we list, too.

**Comment by junosuarez** [comment]
:evergreen_tree: :hammer: :japanese_castle:
